packages/uploadthing/package.json

```diff
@@ -60,13 +60,25 @@
       }
     },
     "./tw": {
-      "import": {
-        "types": "./tw/index.d.ts",
-        "default": "./tw/index.js"
+      "browser": {
```

Comment:
the `import`,`require`,`default` should be outside of `browser`, `browser` should be just another one next to it. Also browser can just be one esm file path since most of the bundler can handle it will, I guess it doesn't need to have a browser cjs bundle

```
{
  import: ../
  require: ../
  default: ../
  browser: "./tw/index.browser.js"
}
```

Reply:
hmm still no output 
![CleanShot 2024-11-28 at 12 21 20@2x](https://github.com/user-attachments/assets/81d5e51b-aa9e-4ec2-828a-1f9533744db6)

Reply:
can you bump bunchee to 5.6.2? This is the latest version and works for me